app/controllers/orga/seasons_controller.rb

```diff
@@ -44,6 +46,22 @@ def destroy
     redirect_to orga_seasons_path, notice: "Season #{@season.name} has been deleted."
   end
 
+  # In dev env only, this offers a setting to switch parts of season to open / close
+  # Application opens the link to the application form; CodingSummer opens the link to students/status_updates
+  def switch_phase
+    return unless Rails.env.development?
+    @season = current_season
+    case params[:option]
+    when 'Application'
+      @season.fake_application_phase
+    when 'CodingSummer'
+      @season.fake_coding_phase
+    when 'RealTime'
+      @season.back_to_reality
```

Comment:
I tried it. If I only change :back_to_reality to :destroy, the ux for the developers would require more work. I can look into that in the upcoming weeks, in between the sprints. Can you live with this as is? For now?